bittensor/_subtensor/subtensor_impl.py

```diff
@@ -661,7 +661,8 @@ def unstake (
         if response.is_success:
             with bittensor.__console__.status(":satellite: Checking Balance on: ([white]{}[/white] ...".format(self.network)):
                 new_balance = self.get_balance( wallet.coldkey.ss58_address )
-                new_stake = bittensor.Balance.from_tao( self.neuron_for_uid( uid = neuron.uid, ss58_hotkey = wallet.hotkey.ss58_address ).stake)
+                block = self.get_current_block()
```

Comment:
This should not need to happen

Reply:
yeah but doesn't work without it? maybe i was out of date?